plugins/com.aptana.ui/src/com/aptana/ui/dialogs/MultipleInputMessageDialog.java

```diff
@@ -273,9 +273,11 @@ else if (firstElement instanceof ComboViewer)
 						}
 					}
 				}
-
+				else if (firstElement instanceof Boolean)
+				{
+					response.put(fieldName, Boolean.TRUE);
```

Comment:
Since the user has pressed YES, there is no need to rely on the firstElement. We definitely need to pass in YES for the field name.